src/views/certifications/components/accordion-with-mdx-content/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { MDXRemote } from 'next-mdx-remote'
+import Accordion from 'components/accordion'
+import Image from 'components/image'
+import { ImageProps } from 'components/image/types'
+import {
+	MdxA,
+	MdxP,
+	MdxTable,
+	MdxInlineCode,
+	MdxOrderedList,
+	MdxUnorderedList,
+	MdxListItem,
+	MdxBlockquote,
+} from 'components/dev-dot-content/mdx-components'
+import { AccordionWithMdxContentProps, AccordionMdxItem } from './types'
+
+function MdxImage({
+	alt,
+	src,
+	title,
+}: Pick<ImageProps, 'alt' | 'src' | 'title'>) {
+	return <Image alt={alt} src={src} title={title} noBorder={true} />
+}
+
+const MDX_COMPONENTS = {
+	a: MdxA,
+	blockquote: MdxBlockquote,
+	p: MdxP,
+	table: MdxTable,
+	img: MdxImage,
+	inlineCode: MdxInlineCode,
+	ul: MdxUnorderedList,
+	ol: MdxOrderedList,
+	li: MdxListItem,
+}
+
+export function AccordionWithMdxContent({
```

Comment:
💭 It might be nice to eventually lift this up to where `Accordion` lives. I could see it being useful elsewhere.